docs/release/release_0_5_4.md

```diff
@@ -0,0 +1,138 @@
+# napari 0.5.4
+
+*Friday, Sep 27, 2024*
+
+We're happy to announce the release of napari 0.5.4!
+
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+For more information, examples, and documentation, please visit our website,
+https://napari.org.
+
+## Highlights
+
+Another release with a lot of bug fixes, but also some (more!) improvements to
+Shapes layer performance ([#7144](https://github.com/napari/napari/pull/7144),
+[#7256](https://github.com/napari/napari/pull/7256)), and a few nice
+usability/quality of life features!
+
+- you can now follow the value under the cursor around in images in 3D view as
+  well as 2D ([#7126](https://github.com/napari/napari/pull/7126))
+- Use standard keyboard shortcuts to zoom in and out ({kbd}`Command/Ctrl+=`,
+  {kbd}`Command/Ctrl+-`) and to zoom-to-fit ({kbd}`Command/Ctrl+0`)
+  ([#7200](https://github.com/napari/napari/pull/7200))
+- you can now save tiff files larger than 4GB
+  ([#7242](https://github.com/napari/napari/pull/7242))
+
+Finally, we're starting some work on tweaking the UI to make it more
+self-consistent, with the ultimate goal of adding functionality such as showing
+common controls when multiple layers are selected, so that, for example, you
+can select multiple layers and adjust all their opacity settings together. As
+the first step for this, the *layer blending* controls have been moved directly
+under the opacity control, so that all per-layer controls are next to each
+other in the UI ([#7202](https://github.com/napari/napari/pull/7202))
+
+Read on for all the changes in this version!
+
+## New Features
+
+- enable get_value_3d for image layers ([#7126](https://github.com/napari/napari/pull/7126))
+- Preserve area of dock widgets between sessions ([#7247](https://github.com/napari/napari/pull/7247))
+
+## Improvements
+
+- Speed up `get_status` for Shapes layer by using bounding boxes ([#7144](https://github.com/napari/napari/pull/7144))
+- Enh: Add zoom options and keybindings to View menu ([#7200](https://github.com/napari/napari/pull/7200))
+- [ux/ui] Put blending controls under opacity slider in layer controls ([#7202](https://github.com/napari/napari/pull/7202))
+- Enable conflicts detection with layer keybindings when changing viewer ones and add group info to conflicts popup ([#7231](https://github.com/napari/napari/pull/7231))
+- Speedup `_is_convex` by avoid calling `np.roll` ([#7256](https://github.com/napari/napari/pull/7256))
+- Sort plugin manifests in alphabetical order for registration ([#7266](https://github.com/napari/napari/pull/7266))
+
+## Performance
+
+- Speed up `get_status` for Shapes layer by using bounding boxes ([#7144](https://github.com/napari/napari/pull/7144))
```

Comment:
maybe also: `- Speedup `_is_convex` by avoid calling `np.roll` ([#7256](https://github.com/napari/napari/pull/7256))`

Reply:
Yup, was missing a label. Will add it now. (Both the line and the label.)